### PR TITLE
Improve Structure for Developer Documentation

### PR DESCRIPTION
**Pull request by h0ngcha0**

### CODE REVIEW

File: scripts/generate-builtin-functions.js [hunk 1]
```diff
@@ -56,27 +56,27 @@ This file is auto-generated with "scripts/generate-builtin-functions.js"
...
-    stream.end()
+  //  const stream = fs.createWriteStream('./docs/dapps/ralph/built-in-functions.md')
+  //const [categories, map] = extractCategory(json)
+  //writeHeader(stream, categories)
+  //categories.forEach(category => {
+  //  stream.write(`## ${category} Functions\n---\n`)
+  //  const functions = map.get(category)
+  //  functions.forEach(f => writeFunction(f, stream))
+  //})
+  //stream.end()
```
Comment (polarker) [inline scripts/generate-builtin-functions.js]: Why this is commented out?

File: docs/dapps/tutorials/_category_.json
Comment (polarker) [inline docs/dapps/tutorials/_category_.json:1]: We could put `Tutorials` at the begiining, before `Ralph`?

**Review by polarker — APPROVED** [review]
Great work 🔥

File: docs/dapps/tutorials/_category_.json
Comment (h0ngcha0) [reply docs/dapps/tutorials/_category_.json:1]: Done :+1:

File: scripts/generate-builtin-functions.js (hunk 1 above)
Comment (h0ngcha0) [reply scripts/generate-builtin-functions.js]: Sorry, not intentional, did it because I was running `npm run build` offline. Reverted.

**Pull request merged**